tests/test_sort_package_xml.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
+import shutil
+
+import pytest
+
+from pre_commit_hooks import sort_package_xml
+
+files = [
+    "package.xml",
+]
+
+cases = []
+for file in files:
+    cases.append((Path(file), 0))
+    cases.append((Path(file), 1))
+
+
+@pytest.mark.parametrize(("target_file", "answer_code"), cases)
+def test(target_file, answer_code, datadir):
+
+    ok_ng = ".ng" if answer_code else ".ok"
```

Comment:
I remember `NG` isn't a correct English word but Japanglish.

Reply:
So how about renaming them to `correct/incorrect`, `right/wrong`, or `valid/invalid`?

Reply:
https://github.com/tier4/pre-commit-hooks-ros/pull/32/commits/9a7b84e5450faa71584a63bb44b5646cd4ddf463